signac/contrib/job.py

```diff
@@ -23,14 +23,17 @@
 
 
 class _sp_save_hook(object):
+    """ """
 
     def __init__(self, *jobs):
         self.jobs = list(jobs)
 
     def load(self):
+        """ """
         pass
 
     def save(self):
+        """ """
         for job in self.jobs:
             job._reset_sp()
```

Comment:
I am not sure what class `_sp_save_hook` does, what should I add in its doc string?

Reply:
Maybe just say "used to override default save and load behavior for synced data structures"? @vyasr @mikemhenry Any other ideas? The entire class is internal, so the comments could be more oriented towards developers instead of users.

Reply:
Definitely agree that the docs should be more for developers. The goal of the `_sp_save_hook` is to force the job's statepoint resetting to happen whenever the statepoint is changed. Resetting a statepoint requires recomputing the hash and moving the folder, which is outside the scope of just what the synced dict does, so that's what this documentation should indicate.

Reply:
thanks for the details, I have added the doc string accordingly.